### ISSUE TITLE
YOLOv8 CLI documentation incorrectly states that save=False is the default configuration

### ISSUE THREAD

**Issue by trevorstr** [ultralytics/ultralytics#9972]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and found no similar bug report.

### YOLOv8 Component

Predict

### Bug

I am using the Docker command below to launch containers, using the stock, unmodified `ultralytics/ultralytics:latest` container image. The containers are filling up my server's disk space because YOLOv8 is recording video content to the container's filesystem, under the `./runs/detect/predict` directory.

According to the CLI documentation, the default setting should be `save=False`. However, if you don't explicitly set this parameter, then the video from the input stream is recorded to the `./runs` child directory.

YOLO v8 CLI Documentation: https://docs.ultralytics.com/modes/predict/#inference-arguments

![image](https://github.com/ultralytics/ultralytics/assets/147647647/d99d2d16-a03e-49c7-a3e9-fabb7c339687)

In order to resolve this bug, either:

- The documentation should be updated to reflect that `save` is functionally `True`
- The CLI parameters should be updated so that `save` is _actually_ `False` by default

### Minimal Reproducible Example

```
docker run --gpus=all --restart=always -it \
  --detach ultralytics/ultralytics \
  bash -c 'curl -LO https://github.com/ultralytics/assets/releases/download/v8.1.0/yolov8l.pt; yolo predict source=rtsp://user:pass@ip_address imgsz=1920 model=yolov8l.pt conf=0.8'
```

### Additional

Thanks for this awesome project! It's incredibly useful, and your team is stellar!

Originally reported this issue here: https://github.com/orgs/ultralytics/discussions/9957

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
Thank you for bringing this to our attention and for your kind words! 🌟 It seems there might be a discrepancy between the expected default behavior and what's actually happening regarding the `save` parameter. We'll look into this issue further to ensure consistency between our documentation and the implemented defaults. In the meantime, if you want to prevent saving the output explicitly, you can add `save=False` to your command, like so:

```bash
docker run --gpus=all --restart=always -it \
  --detach ultralytics/ultralytics \
  bash -c 'curl -LO https://github.com/ultralytics/assets/releases/download/v8.1.0/yolov8l.pt; yolo predict source=rtsp://user:pass @trevorstr imgsz=1920 model=yolov8l.pt conf=0.8 save=False'
```

This should prevent the videos from being saved to the filesystem. We appreciate your patience and feedback as we work to improve YOLOv8 usability and documentation. Stay tuned for updates!

**Comment by glenn-jocher** [comment]
@trevorstr after checking this a bit further there are dual defaults: True for CLI commands like yours, since we assume most users will be running CLI for demonstration purposes and testing, and False when used in Python.

I'll update the documentation to reflect this.

**Comment by glenn-jocher** [comment]
@trevorstr BTW you can eliminate the curl download from your command as YOLOv8l will automatically download from the latest official source if not found locally :)

**Comment by glenn-jocher** [comment]
Great news 😃! Your original issue may now be resolved ✅ in PR #16061. 

To get this update:
- **[Git](https://github.com/ultralytics/ultralytics)** – Run `git pull` from within your `ultralytics/` directory or run `git clone https://github.com/ultralytics/ultralytics` again
- **[Pip](https://pypi.org/project/ultralytics/)** – Update with `pip install -U ultralytics`
- **[Notebooks](https://github.com/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb)** – Check out the updated notebooks  <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/ultralytics/yolov8"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/ultralytics)** – Run `sudo docker pull ultralytics/ultralytics:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and letting us know. Please confirm if this update fixes the issue for you, and don't hesitate to report any other **issues** you find or **feature requests** you may have. Happy training with YOLOv8 🚀!
